worldbank_wdi/match_variables.py

```diff
@@ -0,0 +1,119 @@
+"""Creates a `variable_replacements.json` file of {old variable id} to
+{new variable id} key-value pairs.
+
+`variable_replacements.json` is used in `suggest_chart_revisions.py` for
+determining which charts to update.
+
+Usage:
+
+    python -m worldbank_wdi.match_variables
+"""
+
+import os
+import simplejson as json
+import logging
+from typing import List
+import pandas as pd
+
+from db import get_connection
+from db_utils import DBUtils
+from worldbank_wdi import OUTPATH
+
+UPDATE_DB = False
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+def main():
+    with get_connection().cursor() as cursor:
+        db = DBUtils(cursor)
+        # retrieves old and new datasets
+        df_old_datasets = get_datasets(db=db, new=False)
+        df_new_datasets = get_datasets(db=db, new=True)
+
+        # retrieves old and new variables
+        df_old_vars = get_variables(db=db, dataset_ids=df_old_datasets['id'].tolist())
+        df_new_vars = get_variables(db=db, dataset_ids=df_new_datasets['id'].tolist())
```

Comment:
Likewise with `get_variables`, and you could also probably get rid of `tolist()`